composer.json

```diff
@@ -16,7 +16,7 @@
     "guzzlehttp/ringphp" : "~1.0"
   },
   "require-dev": {
-    "phpunit/phpunit": "^4.7|^5.4",
+    "phpunit/phpunit": "6.3.0",
```

Comment:
The `require-dev` does not affect the user, so it is best to use specific version.